docs/DesignMDIngestion.md

```diff
@@ -0,0 +1,61 @@
+# Generic Metadata Ingestion to MongoDB
+
+## Description
+
+This is the design document discussing the ingestion process of existing metadata
+into Zenko with MongoDB.
+
+The primary use case is to ingest existing metadata from the RING and S3 Connector;
+future use cases will include ingesting metadata from existing storage solutions
+on AWS or Azure.
+
+This specific development will allow Zenko instances to copy and ingest existing
+metadata,
+copying the information from the raft logs completely and allowing MongoDB to be
+a parallel metadata database.
+
+## Requirements
+
+* Copy all existing metadata from pre-existing storage solutions to MongoDB in Zenko.
+
+* Allow MongoDB to be used in parallel with the existing metadata servers with S3
+  Connector.
+
+## Design
+
+The proposed design will be as follows:
+
+* Connect Zenko to the S3 Connector, and Backbeat will connect to each raft session
```

Comment:
Connect Zenko and S3 in which way?
Backbeat will connect to each raft session using what mean? Isn't it already connected to the raft sessions?